tests/lib/rules/max-len.js

```diff
@@ -309,6 +314,17 @@ ruleTester.run("max-len", rule, {
                 }
             ]
         }, {
+            code: "// A comment that exceeds the max comment length, but not the specified max length for code.",
+            options: [40, 4, { comments: 80 }],
```

Comment:
@not-an-aardvark is exactly right :-)

Could we also add a test case where comments have a *shorter* length than code, and the comment length falls in between the comment and code lengths?